01-Back-End Criação de Usuário/CustomersController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace _01_Back_End_Criação_de_Usuário
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomersController : ControllerBase
+    {
+
+        private readonly ICustomersRepository _customers;
+        public CustomersController(ICustomersRepository customers)
+        {
+            _customers = customers;
+        }
+
+        [HttpPost]
+        public virtual ActionResult Create([FromBody] CustomersModel model)
+        {
+            return StatusCode(_customers.Create(model));
+        }
+
+        [HttpGet("Get All Customers")]
+        public ActionResult Get()
```

Comment:
```suggestion
        public IActionResult Get()
```
Seria melhor utilizar da interface IActionResult. Isso vale para todas as outras requisições também